src/System.Runtime.Analyzers/Core/AttributeStringLiteralsShouldParseCorrectly.cs

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
-using Analyzer.Utilities;
+using Microsoft.CodeAnalysis.Semantics;
 
 namespace System.Runtime.Analyzers
 {
     /// <summary>
     /// CA2243: Attribute string literals should parse correctly
     /// </summary>
```

Comment:
Add a comment here explaining that this rule doesn't warn on version like FxCop used to and the reason for the deviation.